### PR TITLE
Update examples for 2.1 by scheduling redraw

### PR DESCRIPTION
**Pull request by pushfoo**
### Changes

tl;dr schedule drawing intervals per [description of breaking changes on Discord](https://discord.com/channels/438622377094414346/477394331112701952/1134825532354216058):
> ...there is a breaking change for 2.1. You need to explicitly scheudle the Window redraw

### ~~I need help with~~ Done

- [x] Testing Windows-specific examples:
     - [x] `python examples/text/advanced_font.py`
     - [x] `python examples/text/font_comparison.py`

- [x] Testing controller input examples (Works according to Ben)
     - [x] `python examples/input/controller.py`
     - [x] `python examples/input/joystick.py`

### Follow-up questions

1. Should we add frame rate constants to make scheduling cleaner?
2. Q: How should we describe the pros and cons of using `schedule` vs `schedule_interval` for readers?
   A: Examples should use `schedule_interval`

### CODE REVIEW

**Comment by benmoran56** [comment]
I made a few comments. Other than those, lets change all instances of `pyglet.clock.schedule` to `pyglet.clock.schedule_interval`.

**Comment by benmoran56** [comment]
I tested the controller and joystick examples on my machine, and they work fine after adding schedule_interval.

**Comment by benmoran56** [comment]
Windows specific examples are tested OK.

**Review by benmoran56 — APPROVED** [review]
